### DISCUSSION TITLE
Stay as close to the Javascript SDK as possible?

### DISCUSSION THREAD

**Discussion by MrCredible** [ramnes/notion-sdk-py#59] [General]
I saw another discussion in this repo about moving away from staying as close to Javascript SDK as possible.
I cant find the discussion or issue where I saw that.

Was there a decision made with that?

I have created a project that creates the various Notion Objects described in the Notion API along with the property types.

I was going to create a new github project, but wanted to avoid that if this repo was going in a more abstracted route and less closely related to the Javascript SDK.

Let me know ! :)

**Comment by ramnes** [comment]
Here is the discussion you're mentioning, I think: https://github.com/ramnes/notion-sdk-py/issues/9

The idea I'm the most confortable with so far is to keep a "notion_client.base" module close to the Javascript SDK and have higher level tools in the root module.

Have a look at PR #10 and #51 that start bringing such kinds of higher level tools.
